src/test/java/com/github/fabienrenaud/jjb/databind/ClientsDeserializationTest.java

```diff
@@ -45,11 +45,5 @@ public void johnzon() throws Exception {
     @Override
     public void qson() throws Exception {
         // FIXME: does not support arrays
-    }    
-    
-    @Override
-    public void fastjson() throws Exception {
-        //FIXME: Dates don't serialize correctly
```

Comment:
Is this fixed?
If not, we need to remove fastjson from `CLIENTS` support in https://github.com/fabienrenaud/java-json-benchmark/blob/master/src/main/java/com/github/fabienrenaud/jjb/support/BenchSupport.java#L50
If it is, please bring back the test.

Reply:
it's fixed, fastjson 2.0.31 support deserialize OffsetDateTime